src/endpoint/s3/ops/s3_post_bucket_delete.js

```diff
@@ -12,7 +12,7 @@ const S3Error = require('../s3_errors').S3Error;
 async function post_bucket_delete(req) {
 
     const quiet = req.body.Delete.Quiet && req.body.Delete.Quiet[0];
-    const objects = _.map(req.body.Delete.Object, obj => ({
+    let objects = _.map(req.body.Delete.Object, obj => ({
```

Comment:
I would first filter and then map

Reply:
You're right in the next comment.
I cannot filter the array before since the limit of 1,000 keys is on the original objects.
I also checked it before I decided where to filter the array.